documentation/modules/ROOT/pages/connectors/postgresql.adoc

```diff
@@ -12,356 +14,156 @@ ifdef::community[]
-
-{prodname} uses PostgreSQL's logical decoding, which uses replication slots.
-Replication slots are guaranteed to retain all WAL required for {prodname} even during {prodname} outages.
-It is important for this reason to closely monitor replication slots to avoid too much disk consumption and other conditions that can happen such as catalog bloat if a replication slot stays unused for too long.
-For more information please see the official Postgres docs on https://www.postgresql.org/docs/current/warm-standby.html#STREAMING-REPLICATION-SLOTS[this subject].
-
-If you are working with a `synchronous_commit` setting other than `on`,
-the recommendation is to set `wal_writer_delay` to a value such as 10 ms to achieve a low latency of change events.
-Otherwise, its default value is applied, which adds a latency of about 200 ms.
-
-[TIP]
-====
-We strongly recommend reading and understanding https://www.postgresql.org/docs/current/static/wal-configuration.html[the official documentation] regarding the mechanics and configuration of the PostgreSQL write-ahead log.
-====
-endif::community[]
-
-[[postgresql-permissions]]
-=== Setting up Permissions
-
-Next, configure a database user who can perform replications.
-
-Replication can only be performed by a database user that has appropriate permissions and only for a configured number of hosts.
-
-In order to give a user replication permissions, define a PostgreSQL role that has _at least_ the `REPLICATION` and `LOGIN` permissions. For example:
-
-[source,sql]
-----
-CREATE ROLE name REPLICATION LOGIN;
-----
-
-[NOTE]
-====
-Superusers have by default both of the above roles.
+It is strongly recommended that you configure a PostgreSQL connector to set `snapshot.mode` to `exported`. The `initial`, `initial only` and `always` modes can lose a few events while a connector switches from performing the snapshot to streaming change event records when a database is under heavy load.
+This is a known issue and the affected snapshot modes will be reworked to use `exported` mode internally (link:https://issues.redhat.com/browse/DBZ-2337[DBZ-2337]).
 ====
 
-Finally, configure the PostgreSQL server to allow replication to take place between the server machine and the host on which the PostgreSQL connector is running:
-
-.pg_hba.conf
-[source]
-----
-local   replication     <youruser>                          trust   // <1>
-host    replication     <youruser>  127.0.0.1/32            trust   // <2>
-host    replication     <youruser>  ::1/128                 trust   // <3>
-----
-<1> Tells the server to allow replication for `<youruser>` locally (i.e. on the server machine)
-<2> Tells the server to allow `<youruser>` on `localhost` to receive replication changes using `IPV4`
-<3> Tells the server to allow `<youruser>` on `localhost` to receive replication changes using `IPV6`
+[id="snapshot-mode-settings"]
+.Settings for `snapshot.mode` connector configuration property
+[cols="20%a,80%a",options="header"]
```

Comment:
We began the move away from using `options="header"` for tables and I think we should continue that here for consistency across all of the documentation.  There are multiple occurrences of this in this file btw.

Reply:
In a table, `options="header" is required downstream because it improves accessibility for visually impaired folks who are hearing the text from a screen reader. 
Why are you moving away from using it?
And can you reverse that decision?

Reply:
I believe this was a decision that came down when the MySQL documentation was first reworked.  We were told that the use of that was redundant and I've since stopped using it when adding new tables to the documentation.  If they should be retained, then we need to review existing documentation for all `[cols=` occurrences and make sure that the options block is restored.

Reply:
I can do that - Should I go ahead and create a DBZ Jira to update doc files so that all tables have the header option?

Reply:
Sure; especially if its critical for downstream in someway.  It's something we can also work toward over time too if its not something that necessarily needs to all be done at once.